python/lsst/cp/pipe/ptc/cpPtcExtract.py

```diff
@@ -515,16 +515,14 @@ def run(self, inputExp, inputDims, taskMetadata, inputPhotodiodeData=None):
                 readNoise2 = dict()
                 meanReadNoise = dict()
 
-                expMetadata1 = expRef1.get(component="metadata")
```

Comment:
I now see that we already had `exp1` and `exp2` loaded so this was just extra i/o.

Reply:
I definitely want to deprecate all `taskMetadata` inputs, so I've added the notice to that connection as well.